AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -315,6 +315,14 @@ - (void)startUpdate {
 
 - (void)requestInstallInformationWith:(NSString *)releaseHash {
 
+  // Browser won't open at start if check for update was not requested or if automatic checks are disabled.
+  if ((self.distributeFlags & MSDistributeFlagsDisableAutomaticCheckForUpdate) == MSDistributeFlagsDisableAutomaticCheckForUpdate) {
```

Comment:
Can we add some getter/helper for this?

Reply:
We are going to use a boolean and get rid of flags.